api/photo_app/models.py

```diff
@@ -34,6 +34,17 @@ def __str__(self):
         return '{} commented on post {}'.format(user.username, post.id)
 
 
+class Follower(models.Model):
```

Comment:
There's something called `unique_together`: https://docs.djangoproject.com/en/2.1/ref/models/options/#unique-together.

The idea of this option is to enforce at the database level that two values when combined must be unique.

We can use unique_together for following_user and followed_user. Let me know what you think about it.